src/engraving/libmscore/stafftype.cpp

```diff
@@ -280,8 +280,6 @@ bool StaffType::isSimpleTabStaff() const
     default:
         return false;
     }
-
-    return false;
 }
```

Comment:
Removing this makes other compilers emit warnings. In my experience the universal fix is:
```c++
    default: 
        break;
    }

    return false;
}
```

Reply:
Done

Reply:
Doesn't even compile though, needs a `break;` in the `default:` case

Reply:
I did write `break;` 😉

Reply:
Outch, reading helps...